metadatastore/commands.py

```diff
@@ -157,6 +157,7 @@ def insert_run_start(time, scan_id, beamline_id, beamline_config, uid=None,
     if project is None:
         project = ''
 
+    beamline_config = BeamlineConfig.objects(uid=beamline_config).first()
```

Comment:
Maybe use `BeamlineConfig.get(uid=beamline_config)` which is marginally faster (maybe?) and will raise if there is any number of results other than 1.